rkt/rkt.go

```
@@ -223,10 +224,23 @@ func main() {
 	// Make help just show the usage
 	cmdRkt.SetHelpTemplate(`{{.UsageString}}`)
 
-	// // Uncomment to update rkt.bash
-	// stdout.Print("Generating rkt.bash")
-	// cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
-	// os.Exit(0)
+	// update rkt.bash
+	if os.Getenv("RKT_GENERATE_BASH_COMPLETION") == "1" {
+		stdout.Print("Generating rkt.bash")
+		cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
+		os.Exit(0)
+	}
+
+	// update rkt.bash
+	if os.Getenv("RKT_GENERATE_MAN_PAGE") == "1" {
+		header := &doc.GenManHeader{
+			Title:   "rkt - App Container runtime",
+			Section: "1",
+		}
+
+		doc.GenManTree(cmdRkt, header, "dist/manpages")
```

Maybe we should not ignore errors?